app/scrapers/exalead.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Exalead(Scraper):
+    """Scrapper class for Exalead"""
+
+    def __init__(self):
+        self.url = 'https://www.exalead.com/search/web/results/'
+        self.defaultStart = 0
+        self.startKey = 'start_index'
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls = []
+        for a in soup.findAll('a', {'class': 'title'}):
+            urls.append({
+                'title': a.getText(),
+                'link': a.get('href')
+                })
+        print('parsed' + str(urls))
```

Comment:
OPTIONAL: Would it make sense to change this line to __print('Exalead parsed: ' + str(urls))__ so that it is easier to know that the results came from the expected scraper?

Reply:
@cclauss i guess it doesn't matter. As it will be on the debuggers side where the print command would work and the scrapper can be known using the api comand whenever a new search is done as it is given in the query parameters.
Cool ?

Reply:
If it does not matter then please explain why #271 was so difficult for us to discover?

Reply:
That was just my opinion @cclauss.
After going through #271 i believe that you are right in saying so.
I apologise for the mistake. :)

Reply:
Agter going througj the code i believe that i might have found the possible fix for #271 as well. I will try to work it out and send a PR  as soon as possible

Reply:
@bhaveshAn It is optional but your thoughts on this one?

Reply:
Now check. :smiley: